delegation-verify-coordinator/values.yaml

```diff
@@ -55,33 +91,40 @@ coordinator:
     #       name: mySecret
     #       key: bar  offline:
 secret:
+  # -- The gcp Service Account
   gcpServiceAccount:
 
 serviceAccount:
   coordinator:
 
-    # Annotations to add to the service account
+    # -- Annotations to add to the service account
     annotations: {}
 
   worker:
 
-    # Annotations to add to the service account
+    # -- Annotations to add to the service account
     annotations: {}
-     
-  # Specifies whether a service account should be created
+
+  # -- Specifies whether a service account should be created
   create: true
-  # Automatically mount a ServiceAccount's API credentials?
+  # -- Automatically mount a ServiceAccount's API credentials?
   automount: true
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
+  # -- The name of the service account to use.
+  # -- If not set and create is true, a name is generated using the fullname template
```

Comment:
nitpick: I think this line is ignored and only the last comment line is picked up. So potentially we either merge the comment lines or remove one of them

Reply:
This should be fixed now